medical_medicament_us/demo/medical_medicament_gcn_demo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="medical_medicament_gcn_us_1" model="medical.medicament.gcn">
+        <field name="name">Demo GCN</field>
```

Comment:
A GCN is a generic code identifier. All drugs have them - it is used to identify the specific formularies of drugs.

I don't know of any public datasets, but I do have one. Let me know the specific drugs you're looking for and I will get for you. It is unique to the Drug and dosage, so for example a Acetaminophen-Hydrocodone 325/10 is the same as a Loratab or Norco 10.